include/eosio/convert.hpp

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
-#include <eosio/for_each_field.hpp>
-#include <eosio/stream.hpp>
+#include "for_each_field.hpp"
+#include "stream.hpp"
```

Comment:
Some eosio include files have been changed to `"` include, I assume to avoid pulling in files from eos repo when integrated. However, this is not done universally in all files. I think we need to stick to one "style". Same goes for `"../include/eosio/`. Also there are a few commented out includes that can be removed.